can/viewer.py

```diff
@@ -353,6 +390,7 @@ def parse_args(args):
         "\n        | ESQ/q   | Exit the viewer         |"
         "\n        | c       | Clear the stored frames |"
         "\n        | s       | Sort the stored frames  |"
+        "\n        | h       | Toggle byte highlights  |"
```

Comment:
Maybe better state that byte *changes* are visualized?

```suggestion
        "\n        | h       | Toggle highlight byte changes |"
```